base/bin/migration.mustache

```diff
@@ -2,9 +2,9 @@ const fs = require('fs');
 const mustache = require('mustache');
 
 const now = new Date().toISOString();
-const migrationName = now + '_' + process.argv[2];
+const migrationName = `${now}_${process.argv[2]}`;
```

Comment:
I thought we'd already changed to node 6? https://github.com/Prismatik/redbeard/blob/master/.travis.yml
I think it's ok to use the latest and greatest isn't it? You're only going to use redbeard for new projects.